projects/ngx-datatable/src/lib/components/body/body-row.component.ts

```diff
@@ -217,4 +222,12 @@ export class DataTableBodyRowComponent<TRow = any> implements DoCheck, OnChanges
   onTreeAction() {
     this.treeAction.emit();
   }
+
+  /** Returns the row index, or if in a group, the index within a group. */
+  private get innerRowIndex(): number {
+    if (!this.rowIndex) {
+      return 0;
+    }
+    return this.rowIndex.indexInGroup ?? this.rowIndex.index;
+  }
```

Comment:
Nit: Any reason you explicitly check for `this.rowIndex` here and not just do this all inline?

```suggestion
  private get innerRowIndex = (): number => this.rowIndex?.indexInGroup ?? this.rowIndex?.index ?? 0;
```

Reply:
changed, but sticked to the normal function notation to stay inline with all other set/get.